regml.py

```diff
@@ -11,6 +11,7 @@
 
 import click
 from lxml import etree
+from lxml import objectify
```

Comment:
If we're switching to lxml's `objectify` API, do we still need the `etree` import?